changelog/fragments/upgrade-kb.yaml

```
@@ -0,0 +1,57 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      Upgrade controller-runtime (sigs.k8s.io/controller-runtime) dependency version
+      from `v0.6.2` to `v0.6.3`. More info: https://github.com/kubernetes-sigs/controller-runtime/releases/tag/v0.6.3
+
+    # kind is one of:
+    # - addition
+    # - change
+    # - deprecation
+    # - removal
+    # - bugfix
+    kind: "change"
+
+    # Is this a breaking change?
+    breaking: false
+
+    # Migration can be defined to automatically add a section to
+    # the migration guide. This is required for breaking changes.
+    migration:
+      header: Upgrade controller-runtime version
```

However unlikely, we might bump this version twice in between two minor versions, so the title should include which version the user should bump to
```suggestion
      header: Upgrade sigs.k8s.io/controller-runtime version to v0.6.3
```

---

done